File: executable/julia/labelled-heatmap.jl
using Plotly

using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "z" => [1 20 30; 20 1 60; 30 60 1; 50 80 -10; 1 30 20], 
    "x" => ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"], 
    "y" => ["Morning", "Afternoon", "Evening"], 
    "type" => "heatmap"
  ]
]

response = Plotly.plot([data], ["filename" => "labelled-heatmap", "fileopt" => "overwrite"])
plot_url = response["url"]

File: executable/python/labelled-heatmap.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

data = Data([
    Heatmap(
        z=[[1, 20, 30, 50, 1], [20, 1, 60, 80, 30], [30, 60, 1, -10, 20]],
        x=['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
        y=['Morning', 'Afternoon', 'Evening']
    )
])

plot_url = py.plot(data, filename='labelled-heatmap')